test/jdk/tools/jpackage/helpers/jdk/jpackage/test/Executor.java

```diff
@@ -90,6 +91,15 @@ public Executor setRemovePath(boolean value) {
         return this;
     }
 
+    public Executor setWinEnableUTF8(boolean value) {
```

Comment:
This function aims to make a to-be-executed command produce output in English. To achieve this goal implementation will call `chcp 65001`. The implementation may change, but the name of the function should remain the same. That said I suggest giving it a name that reflects its purpose and not the implementation. Something like: `setWinRunWithEnglishLocale()`

Reply:
> The implementation may change, but the name of the function should remain the same.

That certainly makes sense. I changed setWinEnableUTF8 to setWinRunWithEnglishOutput as you suggested.